apps/extension/src/background/keyring/service.ts

```diff
@@ -423,12 +479,6 @@ export class KeyRingService {
       await this.broadcaster.updateBalance();
     } catch (e) {
       console.warn(e);
-      await this.broadcaster.completeTx(
```

Comment:
Why does the event for tx failure get removed here?

Reply:
Looks like I removed it by accident :D Thanks!